openstack/identity/v3.0/users/urls.go

```diff
@@ -15,3 +15,11 @@ func updateURL(client *golangsdk.ServiceClient, userID string) string {
 func getURL(client *golangsdk.ServiceClient, userID string) string {
 	return client.ServiceURL(rootPath, "users", userID)
 }
+
+func updateLoginProtectURL(client *golangsdk.ServiceClient, userID string) string {
```

Comment:
how about merge `updateLoginProtectURL` and `getLoginProtectURL` to `loginProtectURL`?